app/controllers/carts_controller.rb

```diff
@@ -1,5 +1,24 @@
 # frozen_string_literal: true
 
 class CartsController < ApplicationController
+  before_action :order_finder, only: %i[show show_order]
+  before_action :show_filter, only: %i[index show]
+
+  def index
+    @orders = OrderSorter.call(params, current_customer.carts).reverse
+  end
+
   def show; end
+
+  def show_order; end
```

Comment:
not RESTful. This should be `OrdersController#show`

Reply:
Moved to ```OrdersController#show``` as ```show```